crates/transaction-emitter-lib/src/emitter/account_minter.rs

```diff
@@ -591,7 +592,25 @@ pub async fn execute_and_wait_transactions(
                 None,
             )
             .await
-            .map_err(|e| format_err!("Failed to wait for transactions: {:?}", e))?;
+            .map_err(|e| {
+                warn!(
+                    "Failed to wait for transactions: {:?}, txn: {:?}. [{:?}] We were submitting transactions for account {}: from {} - {}, now at {}",
+                    e,
+                    txn,
+                    client.path_prefix_string(),
+                    account.address(),
+                    start_seq_num - state.txns.len() as u64,
+                    start_seq_num,
+                    account.sequence_number()
+                );
+
+                // sleep so that counters on Grafana get updated.
+                std::thread::sleep(Duration::from_secs(30));
```

Comment:
Out of curiosity: 
* Why do we need this? Is it because we often submit batch load in sequence and so it's difficult to distinguish in the graphs when there's failure?

LGTM, but I want to make sure we're not making the test run too long  with these sleep 30s in the worst case

Reply:
if we reach this failure point, this ends the test.

Issue is that for debugging we need Grafana counters, and they might not get updated after this point, as test ends soon after.

Reply:
Made the comment inline much more clearer

Reply:
I see, thanks for the explanation! Comment makes it a lot clearer :)